src/mongo_client.ts

```diff
@@ -32,6 +32,32 @@ import type { SrvPoller } from './sdam/srv_polling';
 import type { Connection } from './cmap/connection';
 import type { LEGAL_TLS_SOCKET_OPTIONS, LEGAL_TCP_SOCKET_OPTIONS } from './cmap/connect';
 
+/** @public */
+export const LogLevel = {
+  error: 'error',
+  warn: 'warn',
+  info: 'info',
+  debug: 'debug'
+} as const;
+
+/** @public */
+export const ServerApiVersion = {
+  v1: '1'
+};
+
+/** @public */
+export type ServerApiVersionId = typeof ServerApiVersion[keyof typeof ServerApiVersion];
+
+/** @public */
+export interface ServerApi {
```

Comment:
The spec calls for this to be a class, but I didn't think it would be idiomatic in Node to make users import the ServerApi class and pass a `new ServerVersion(...)` in the constructor, versus accepting a plain object and providing a TypeScript interface. Does this make sense, or should it be a class like the spec advises?

Reply:
I agree about it being idiomatic for nodejs
I guess just to consider an alternative we could make a class similar to our ReadPreference where we have static predefined instances of it? looking something like

```ts
new MongoClient('...', { serverApi: ServerApi.V1 })
```

Just putting this out there to say we considered it but I think avoiding an import is more satisfying for users.